doc/howto/cluster_manage.md

```diff
@@ -73,26 +73,41 @@ When the evacuated server is available again, use the `lxc cluster restore` comm
 This command also moves the evacuated instances back from the servers that were temporarily holding them.
 
 (cluster-manage-delete-members)=
-## Deleting cluster members
+## Delete cluster members
 
-To cleanly delete a member from the cluster, use the `lxc cluster remove` command.
+To cleanly delete a member from the cluster, use the following command:
 
-You can only delete members that are online and that don't have any instances located on them.
-If required, you can force-remove a cluster member (for example, if the server is permanently offline or had to be reinstalled).
-To do so, enter the following command:
+    lxc cluster remove <member_name>
+
+You can only cleanly delete members that are online and that don't have any instances located on them.
+
+### Deal with offline cluster members
+
+If a cluster member goes permanently offline, you can force-remove it from the cluster.
+Make sure to do so as soon as you discover that you cannot recover the member.
+If you keep an offline member in your cluster, you might encounter issues when upgrading your cluster to a newer version.
+
+To force-remove a cluster member, enter the following command:
 
     lxc cluster remove --force <member_name>
```

Comment:
@ru-fu should we make clear that this command must be run from one of the online members of the cluster?
Perhaps its obvious, but if the offline member is still operational, but just permanently unreachable from the rest of the cluster, then it may be the user tries to run it on the offline member.